p4-16/spec/P4-16-spec.mdk

```diff
@@ -3194,8 +3194,35 @@ casts `a`, which was initialized to `E.e2` to a `bit<8>`, using the specified
 fixed-width unsigned integer representation for `E.e2`, `1`.  The variable `b` is then set to the
 symbolic value `E.e2`, which corresponds to the fixed-width unsigned integer value `1`.
 
-Because it is always safe to cast from an `enum` to its underlying fixed-width integer type,
-implicit casting from an `enum` to its fixed-width (signed or unsigned) integer type is also supported (see Section [#sec-implicit-casts]):
+We can apply a bit-slice operator on an `enum` with an underlying type 
```

Comment:
this is a long description, wouldn't it be shorter to say that this is equivalent to casting the enum to the underlying type and then applying the slice?